config/base.js

```diff
@@ -63,7 +63,7 @@ const SCHEMA_SUBSCRIBE = Joi.object().keys({
     )
 });
 
-const SCHEMA_CONFIG = Joi.object()
+const SCHEMA_MERGE_CONFIG = Joi.object()
```

Comment:
Consider renaming. Just having `MERGE` could be confusing. It is not clear whether it is applied before merge, after merge and during merge, etc
```suggestion
const SCHEMA_CONFIG_PRE_TEMPLATE_MERGE = Joi.object()
```